zilliqa/src/consensus.rs

```diff
@@ -162,9 +162,6 @@ pub struct Consensus {
     /// Receipts cache
     receipts_cache: HashMap<Hash, (TransactionReceipt, Vec<Address>)>,
     receipts_cache_hash: Hash,
-    /// Cache stake/stakers
-    stake_cache: BTreeMap<u64, u128>,
-    stakers_cache: BTreeMap<(u64, Hash), Vec<NodePublicKey>>,
```

Comment:
Unresolved comment to ensure @shawn-zil has a chance to test my assertion that this change doesn't hurt performance before we merge it.

Reply:
The initial numbers are good:
- Original 25 validators: 2.47s
- PR 1792 (Shawn) - 0.51s
- PR 1813 (James) - 0.34s

So, I've assigned this PR to close the issue. Thanks!